### ISSUE TITLE
dm scrollbar background

### ISSUE THREAD

**Issue by cmdtruth** [refact0r/midnight-discord#67]
Hi refact0r, 

Thanks for a wonderful theme. I recently installed it however I am facing some alignment issues with the theme. The chat bubble as shown below on the right side is misaligned. I have tried to mark it to give it better visibility. The blue outline shows the misalignment and the red line on the left shows it perfectly aligned.
<img width="443" alt="issue_outlined" src="https://github.com/refact0r/midnight-discord/assets/142883094/5fc3cc38-9b4e-4279-b6d1-50a3632640eb"> <img width="443" alt="issue1" src="https://github.com/refact0r/midnight-discord/assets/142883094/70dd3581-9111-4db0-99dd-c063b2466869">

I have not changed any configurations given in the file and it seems to be case with the main theme alignment.

**Comment by refact0r** [comment]
what client are you using? i do not see anything on my end. make sure you have the most recent version of the theme file.

**Comment by leorium-XL** [comment]
Yea, I don't see that on my end either. Might be a lil blind, but it looks like it's the scrollbar that's causing the problem? Check if the problem persists with other flavors of this theme, it might just be your Discord client buggin'

**Comment by cmdtruth** [comment]
I was using the web app and it seems to be due to the scrollbar that's causing the problems. That's why the problem is only on the left side. Thanks for taking a look refact0r and leorium-XL. I will try to think of a workaround or just accept my fate lol.

**Comment by leorium-XL** [comment]
Hmm, it's probably a browser issue; I get the same scrollbar issue on Firefox. What browser are you using? I would try to see if the problem occurs on another browser with the same theme file.

**Comment by cmdtruth** [comment]
Yes I think it occurs on chromium too since I have tried Firefox and Brave too for discord. Maybe the theme is a bit misaligned on browsers due to the scroll wheel.

**Comment by refact0r** [comment]
is this still happening?
